includes/misc-functions.php

```diff
@@ -394,14 +394,11 @@ function give_get_purchase_session() {
  */
 function give_payment_gateway_donation_summary( $donation_data, $name_and_email = true, $length = 255 ) {
 
-	$summary = '';
-
 	$form_id = isset( $donation_data['post_data']['give-form-id'] ) ? $donation_data['post_data']['give-form-id'] : '';
 
 	// Form title.
-	if ( isset( $donation_data['post_data']['give-form-title'] ) ) {
-		$summary .= $donation_data['post_data']['give-form-title'];
-	}
+	$summary = ( -! empty( $donation_data['post_data']['give-form-title'] ) ? $donation_data['post_data']['give-form-title'] : __( 'Untitled donation form', 'give' ) );
```

Comment:
What is the hypen doing before `-!` in this check?

Reply:
Also rather than saying "Untitled donation form" I like your idea of saying "Donation form ID #123"

Reply:
@DevinWalker 

1> What is the hypen doing before -! in this check?
-> Don't Know why I have added that May be I would be in some another world 

2> Also rather than saying "Untitled donation Form" I like your idea of saying "Donation form ID #123"
-> Add ( Do you want me to add a link to that form ID as well )

Reply:
1. Haha. Nah not another world!
2. No link is necessary but please change it from "Untitled donation form"

Reply:
@DevinWalker 

Done